src/crud.js

```diff
@@ -0,0 +1,61 @@
+const tasklistDisplay = document.querySelector('.shores');
+const inputValu = document.querySelector('.newTask');
+const addBtn = document.querySelector('.addtolist');
+
+let taskList = [];
+
+const display = () => {
+  tasklistDisplay.innerHTML = '';
+  taskList = JSON.parse(localStorage.getItem('localItem')) || [];
```

Comment:
- [ ] Kindly, make sure that you implement well your localStorage. For we can only save the first item recorded in the localStorage but others are not saved in it as you can see in the figure below:
![image](https://user-images.githubusercontent.com/71874425/177428985-dd05c8f4-5b2f-48df-a6b9-abfac69b7e4b.png)
Make sure that you solve this issue. then,
- [ ] Implement your project, so that all changes in your To-Do List be saved in your localStorage.

Reply:
![Screenshot from 2022-07-06 00-06-25](https://user-images.githubusercontent.com/100702612/177431420-16fa725a-acdb-4d8b-b1c3-cc08f052dc7a.png)
@RolandM99  please check again storage is working fine .thanks for resolving this issue.